.github/workflows/ci.yml

```diff
@@ -201,6 +201,38 @@ jobs:
           docker_registry: ${{ inputs.docker_registry }}
           docker_username: ${{ inputs.docker_username }}
           docker_password: ${{ secrets.docker_password }}
+          deploy_annotate_pr: ${{ inputs.deploy_annotate_pr }}
+          pull_request_number: ${{ inputs.pull_request_number }}
+      
+      # - name: Read data from file
+      #   if: matrix.check == 'compose:test'
+      #   id: read_data
+      #   run: |
+      #     ls
+      #     cov_sum=$(cat app/output_directory/coverage-summary.txt)
+      #     echo "::set-output name=cov_data::$cov_sum"
+      #     echo $cov_sum
+      #   shell: bash
+      
+      - name: Code Coverage Report
+        if: matrix.check == 'compose:test'
+        uses: irongut/CodeCoverageSummary@v1.3.0
+        with:
+          filename: app/output_directory/coverage.cobertura.xml
```

Comment:
@Patel-world  - I had mentioned earlier during the implementation as well - We should not be performing any checks against the command / scheme used for generating coverage report.

Instead we should be checking against a file name as it can be consistent amongst coverage reports.

We can perform check against - `app/output/coverage.xml`.  (Changing `output_directory` to `output`).
Is it possible?